YoutubeDownloader/Views/Components/DashboardView.xaml.cs

```diff
@@ -1,23 +1,25 @@
-﻿using System.Windows.Input;
+using Avalonia.Input;
+using Avalonia.Interactivity;
 
 namespace YoutubeDownloader.Views.Components;
 
-public partial class DashboardView
+public partial class DashboardView : UserControlBase
 {
     public DashboardView()
     {
         InitializeComponent();
+        QueryTextBox.AddHandler(InputElement.KeyDownEvent, OnQueryTextBoxKeyDown, RoutingStrategies.Tunnel);
```

Comment:
Maybe could inline the `OnQueryTextBoxKeyDown` method as a delegate?

Reply:
done